src/groupchat.c

```diff
@@ -598,6 +836,55 @@ static void groupchat_onKey(ToxWindow *self, Tox *m, wint_t key, bool ltr)
     }
 }
 
+static void draw_peer(ToxWindow *self, Tox *m, ChatContext *ctx, uint32_t i)
+{
+    pthread_mutex_lock(&Winthread.lock);
+    const uint32_t peer_idx = i + groupchats[self->num].side_pos;
+    const uint32_t peernum = groupchats[self->num].name_list[peer_idx].peernum;
+    const bool is_self = tox_conference_peer_number_is_ours(m, self->num, peernum, NULL);
+    const bool audio = groupchats[self->num].audio_enabled;
+    pthread_mutex_unlock(&Winthread.lock);
+
+    /* truncate nick to fit in side panel without modifying list */
+    char tmpnck[TOX_MAX_NAME_LENGTH];
```

Comment:
tmpnick? It took me a few seconds to figure out what "nck" means (is this a common abbreviation elsewhere in toxic?).